packages/Select/src/select.scss

```diff
@@ -0,0 +1,103 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-select-font-family: #{tokens.$primary-font-family};
+  --ids-select-font-size: #{tokens.$font-size-4};
+  --ids-select-font-weight: #{tokens.$font-weight-regular};
+  --ids-select-line-height: #{tokens.$space-4};
+  --ids-select-height: #{tokens.$space-6};
+  --ids-select-margin: #{tokens.$space-1} #{tokens.$space-0};
+  --ids-select-color: #{tokens.$grey-800};
+  --ids-select-chevron: #{tokens.$grey-800};
+  --ids-select-background: #{tokens.$samoyed};
+  --ids-select-border-color: #{tokens.$grey-300};
+  --ids-select-border-radius: #{tokens.$border-radius-sm};
+  --ids-select-number-of-option-to-display: 4;
+
+  /* Compact */
+  --ids-select-font-size-compact: #{tokens.$font-size-3};
+  --ids-select-line-height-compact: #{tokens.$line-height-xxl};
+  --ids-select-height-compact: #{tokens.$space-5};
+
+  /* Active */
+  --ids-select-border-color-active: #{tokens.$electric-blue-500};
+
+  /* Disabled */
+  --ids-select-color-disabled: #{tokens.$grey-500};
+  --ids-select-background-disabled: #{tokens.$grey-200};
+  --ids-select-border-color-disabled: #{tokens.$grey-200};
+
+  /* Error */
+  --ids-select-border-color-error: #{tokens.$coral-500};
+}
+
+.ids-select {
+  font-size: var(--ids-select-font-size);
+  font-family: var(--ids-select-font-family);
+  font-weight: var(--ids-select-font-weight);
+  line-height: var(--ids-select-line-height);
+  height: var(--ids-select-height);
+  width: 100%;
+  padding: 0 var(--ids-select-padding);
+  position: relative;
+  background-color: var(--ids-select-background);
+  border: 1px solid var(--ids-select-border-color);
+  border-radius: var(--ids-select-border-radius);
+  box-sizing: border-box;
+  appearance: textfield;
```

Comment:
This property must be used on form elements otherwise it has on board effects for example in Safari

<img width="579" alt="safari-render" src="https://user-images.githubusercontent.com/2379339/186970051-3546a6b7-3552-40c1-b738-193c3d06e40c.png">